examples/generative/speaker_recognition_using_cnn.py

```diff
@@ -0,0 +1,540 @@
+"""
+Title: Speaker Recognition Using a Convolutional Network
+Author: [Fadi Badine](https://twitter.com/fadibadine)
+Date created: 14/06/2020
+Last modified: 28/06/2020
+Description: Classify speakers using Fast Fourier Transform (FFT) and a 1D Convnet.
+"""
+"""
+## Introduction
+
+This example demonstrates how to create a model to classify speakers from the
+frequency domain representation of speech recordings, obtained via Fast Fourier
+Transform (FFT).
+
+It shows the following:
+
+- How to use `tf.data` to load, preprocess and feed audio streams into a model
+- How to create a 1D convolutional network with residual
+connections for audio classification.
+
+Our process:
+
+- We prepare a dataset of speech samples from different speakers, with the speaker as label.
+- We add background noise to these samples to augment our data.
+- We take the FFT of these samples.
+- We train a 1D convnet to predict the correct speaker given a noisy FFT speech sample.
+
+Note:
+
+- This example should be run with TensorFlow 2.3 or higher, or `tf-nightly`.
+- The noise samples in the dataset need to be resampled to a sampling rate of 16000 Hz
```

Comment:
Thanks for the review!

As discussed in https://github.com/keras-team/keras-io/pull/99#discussion_r443256765, I did those offline using ffmpeg command line. On colab I use !ffmpeg

Reply:
But since the script assumes we have a dataset downloaded from Kaggle and then splits it into audio and noise folder, maybe it makes more sense now to resample as part of the python script.
Otherwise, the code will ignore any noise that is not at 16000 while the user might not have noticed that resampling is needed beforehand.
What do you think? Do you have any suggestion on what to use? 
Thanks!

Reply:
Please include the resampling. It doesn't matter if it's command line code or Python code. You can put command lines in your script by using a shell block (see tutobooks docs)

Reply:
Ok. Will do that.
Thanks!

Reply:
Command line included